source/usr/lib/mcvirt/parser.py

```diff
@@ -125,6 +125,40 @@ def __init__(self, verbose=True, auth_object=None):
             metavar='Target user',
             help='The user to change the password of'
         )
+        self.create_user_subparser = self.user_subparser.add_parser(
+            'create',
+            help='Create a new user',
+            parents=[self.parent_parser]
+        )
+        self.create_user_subparser.add_argument(
+            'new_username',
+            metavar='User',
+            type=str,
+            help='The new user to create'
+        )
+        self.create_user_subparser.add_argument(
```

Comment:
I would create a mutual_exclusive_group here to ensure that --generate-password and --user-password cannot both be passed